### PR TITLE
Add adjutant

### PR DESCRIPTION
**Pull request by holmsten**
This PR enable you to deploy Adjutant 0.5.0 in a Train environment.

### CODE REVIEW

File: ansible-role-requirements.yml [hunk 1]
```diff
@@ -318,3 +318,8 @@
   scm: git
   version: 546253abdd967c9d8abddcd1b7cb2310f6ed6a58
   trackbranch: stable/train
+- name: os_adjutant
+  scm: git
+  src: https://github.com/elastx/openstack-ansible-os_adjutant
+  version: 39fe84b594fed13d32b514f43f15642059d12c37
+  trackbranch: stable/train
```
Comment (Brittlejf) [inline ansible-role-requirements.yml]:
```suggestion
  trackbranch: elastx/train
```

Comment (Brittlejf) [reply ansible-role-requirements.yml]: Can't see a stable/train branch in our or the upstream project so I assume we should use elastx/train.

**Review by Brittlejf — APPROVED** [review]
Found one minor error otherwise LGTM

**Review by huxcrux — APPROVED** [review]
LGTM

**Pull request merged**